### ISSUE TITLE
New brush manager

### ISSUE THREAD

**Issue by dacap** [aseprite/aseprite#3219]
Issue created from #1387 and other comments:

- [ ] New brushes manager (https://github.com/aseprite/aseprite/issues/175) with [preset brushes and patterns](https://twitter.com/cyangmou/status/818191210732994566)
- [ ] Possibility to Import/Export brushes
- [ ] Named brushes
- [ ] Import GG brushes ([1bpp bmps](https://twitter.com/not_surt/status/818200706255437824))
- [ ] Brushes should be in extensions (https://github.com/aseprite/aseprite/issues/1403)

Basically the brush popup must be re-designed to simplify the management of brushes (better organization, simplify saving/loading brushes, naming brushes, switching between brushes and assigning keyboard shortcuts to brushes).
